docs/haskell.rst

```diff
@@ -25,9 +25,26 @@ In this tutorial you'll learn how to:
 Before you begin
 ----------------
 
-To prepare for the tutorial, first `install Bazel`_ if you don't have
-it installed already. Then, retrieve the ``rules_haskell`` GitHub
-repository::
+On a Unix system you will need the following tools installed.
+
+* ``gcc``
+* ``libffi``
+* ``libgmp``
+* ``libtinfo5``
+* ``make``
+* ``python3`` (``python`` also needs to be available in ``$PATH``, depending on your distro, this might require ``python2`` too)
```

Comment:
```suggestion
* ``python3`` (``python`` also needs to be available in ``$PATH``, depending on your distro, this might require installing  the ``python`` meta-package too (which might be Python 2 or 3, ``rules_haskell`` works with both)
```

Reply:
There was a double parenthesis issue, so I've addressed it in another commit https://github.com/tweag/rules_haskell/pull/1162/commits/e335753457d08aa6204cd5a1ad70d56cd31cbc5e